tests/services/deduplication/test_dedup_extractor_xxe.py

```
@@ -72,3 +73,34 @@ def test_extract_odt_handles_malformed_xml(tmp_path: Path) -> None:
+        odt,
+        f'<?xml version="1.0"?><office:document-content {_NS}>'
+        "<office:body><office:text>"
+        "<text:p>Hello without defusedxml</text:p>"
+        "</office:text></office:body></office:document-content>",
+    )
+
+    original_import = builtins.__import__
+
+    def fail_defusedxml_import(
+        name: str,
+        globals: dict[str, object] | None = None,
```

Ruff (0.15.20)

[error] Function argument `globals` is shadowing a Python builtin

(A002)